src/granite/query/assemblers/sqlite.cr

```diff
@@ -0,0 +1,8 @@
+module Granite::Query::Assembler
+  class Sqlite(Model) < Base(Model)
+    def add_parameter(value : DB::Any) : String
+      @numbered_parameters << value
+      "?"
+    end
```

Comment:
The semantics of this are a bit strange (I'm pretty sure that's my fault, not yours). This method shifts a value onto a list, and then returns a different value which ends up in the query. Is there a better way that can be arranged?

Reply:
Yeah, maybe we just rename this method?  something like "replace_the_parameter_in_the_query_and_store_the_value_for_later_use_to_prevent_sql_injection`  ;-)